ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -541,6 +541,19 @@ tests:
       TEST_SUITE: experimental/reliability/minimal
     workflow: openshift-e2e-openstack-ipi
   timeout: 6h0m0s
+- as: e2e-openstack-ovn-capi
```

Comment:
Does this job need `ovn` in the name? Seems confusing as it's simply the default.

Reply:
It does, that's just an OpenShift-wide convention semi-enforced with automation... which means that if you don't add `ovn`, something will break somewhere and you won't know why